api/src/test/java/org/pmiops/workbench/profile/ProfileServiceTest.java

```diff
@@ -36,7 +43,12 @@
   @Autowired VerifiedInstitutionalAffiliationMapper verifiedInstitutionalAffiliationMapper;
 
   @TestConfiguration
-  @MockBean({FreeTierBillingService.class})
+  @MockBean({
+      FreeTierBillingService.class,
+      InstitutionDao.class,
+      InstitutionService.class,
+      UserService.class,
+  })
```

Comment:
I think I want to move the tests for `verifyInstitutionalAffiliation` into this test class since that function is now in the Service layer